src/commands/createFunction/FunctionCreateStepBase.ts

```diff
@@ -53,6 +56,15 @@ export abstract class FunctionCreateStepBase<T extends IFunctionWizardContext> e
         await verifyExtensionBundle(context, template);
 
         const cachedFunc: ICachedFunction = { projectPath: context.projectPath, newFilePath, isHttpTrigger: template.isHttpTrigger };
+        if (context.functionTemplate?.defaultFunctionName === 'ServiceBusQueueTrigger' || context.functionTemplate?.defaultFunctionName === 'BlobTrigger' || context.functionTemplate?.defaultFunctionName === 'QueueTrigger') {
+            const hostFilePath: string = path.join(context.projectPath, hostFileName);
+            const hostJson = await AzExtFsExtra.readJSON<IHostJsonV2>(hostFilePath);
```

Comment:
It's not likely, but you should still ensure the file is there before you try to do a read/write action on it. People could have created a project and deleted their `host.json` during the wizard or something ridiculous like that.